_data/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I think we can delete this file?

Reply:
I'd like to keep this for now with a view to deleting at a later stage if nothing is added?